spec/solr/params_spec.rb

```diff
@@ -11,14 +11,14 @@
       expect(params.to_h).to include(fq: ['animals:(dogs\ \+\ cats)'])
     end
 
-    it 'properly handles and escapes multiple values and defaults to OR-ing them together' do
+    it 'properly handles and escapes multiple values and defaults to match function' do
       params.fq('animals', ['dogs + cats', 'llamas', 'alpacas'])
       expect(params.to_h).to include(fq: ['animals:(dogs\ \+\ cats OR llamas OR alpacas)'])
     end
 
-    it 'properly handles and escapes multiple values with the :and param passed, AND-ing them together' do
-      params.fq('animals', ['dogs + cats', 'llamas', 'alpacas'], :and)
-      expect(params.to_h).to include(fq: ['animals:(dogs\ \+\ cats AND llamas AND alpacas)'])
+    it 'properly handles and escapes multiple values with the match param passed, OR-ing them together' do
+      params.fq('animals', ['dogs + cats', 'llamas', 'alpacas'], 'contains')
+      expect(params.to_h).to include(fq: ['animals:(*dogs\ \+\ cats* OR *llamas* OR *alpacas*)'])
```

Comment:
With this change, it seems like we're no longer testing our AND-ing ability anywhere.

Reply:
Remember when I was asking about multiple values and filters? I don't think there is an API for it, I think it only existed with the tests. Am I wrong?

Reply:
Right.  Looking at the code here reminded me that we added that to support a future feature, but no current API exists.

Reply:
"Find me all items in collection x OR project y"
vs.
"Find me all items with name a AND name b"